common/changes/@itwin/tree-widget-react/Fix_invert_button_2023-04-18-16-20.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/tree-widget-react",
+      "comment": "Categories tree: fix invert button behavior.",
```

Comment:
Please clarify what was wrong with the behavior / how it was fixed.

Reply:
Fixed